pkg/apis/defaults/readinessprobe.go

```
@@ -0,0 +1,34 @@
+package defaults
+
+import (
+	"os"
+	"strconv"
+)
+
+// Environment variables that define default readiness probe parameters for containers
+const (
+	OperatorReadinessProbeInitialDelaySeconds = "RADIXOPERATOR_APP_READINESS_PROBE_INITIAL_DELAY_SECONDS"
+	OperatorReadinessProbePeriodSeconds       = "RADIXOPERATOR_APP_READINESS_PROBE_PERIOD_SECONDS"
+	ReadinessProbeInitialDelaySecondsDefault  = 5
+	ReadinessProbePeriodSecondsDefault        = 10
```

It's probably best to not have default values; this will mask error scenarios. Just let it blow up.